date_range/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Date Range",
     "summary": "Manage all kind of date range",
-    "version": "11.0.2.0.1",
+    "version": "11.0.3.0.1",
```

Comment:
the way I see it, this should be:

```suggestion
    "version": "11.0.3.0.0",
```

Reply:
@mreficent We must use the ocabot merge command to bump the version into the manifest

Reply:
Ok, no problem, but then it should have `11.0.2.0.1` here.